src/util/retry.rs

```diff
@@ -169,11 +169,12 @@ mod test {
         let mut tmp_buf = [0u8; 3];
         let e = retry_reader.read_exact(&mut tmp_buf);
         let e_str = format!("{:?}", &e);
-        assert!(e.is_err(), e_str);
+        assert!(e.is_err(), "{}", e_str);
```

Comment:
can simplify here

Reply:
Addressed in 93a7c393069ca4420433e03338a6534bee19dc8f